Styles.js

```diff
@@ -1545,6 +1545,14 @@ export default StyleSheet.create({
     fontWeight: 'bold',
   },
   flex1: {
-    flex: 1,
-  },
+  flex: 1,
+  },
+connectionPanelSection: {
+  paddingHorizontal: 40,
+},
+connectionPanelTitle: {
+  paddingVertical: 10,
+  borderBottomWidth: 1,
+  borderBottomColor: '#CCCCCC',
```

Comment:
Remove this color from `Styles.js`. Should use the appropriate color in `themes`. In this case I see there is no matching theme style here. Create a new style in both `ThemeWhite.js` and `ThemeBlack.js`:

```js
lightGrayBorderBottom: {
  borderBottomColor: Colors.lightGrey,
}
```